client/react/frontpage/components/SocialPanel.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import './SocialPanel.scss';
+
+// update social media links here
+// img is the base name of the Font Awesome icon
+const socialMediaLinks = [
+  {
+    img: 'facebook',
+    link: 'https://www.facebook.com/uclaradio',
+  },
+  {
+    img: 'twitter',
+    link: 'https://twitter.com/UCLAradio',
+  },
+  {
+    img: 'instagram',
+    link: 'https://www.instagram.com/uclaradio/',
+  },
+  {
+    img: 'snapchat-ghost',
+    link: 'https://www.snapchat.com/add/uclaradio',
+  },
+];
+
+const SocialPanel = React.createClass({
+  convertToFaName(img, size) {
+    return 'fa fa-' + img + ' fa-' + size;
+  },
+  render() {
+    const self = this;
```

Comment:
Why did you declare `self` here vs just using `this.convertToFaName` in line 38?

Reply:
Oops, was just confused about binding in React. Fixed.